Source Main 5.2/source/GMBattleCastle.cpp

```diff
@@ -341,33 +341,46 @@ namespace battleCastle
         if (gMapManager.InBattleCastle() == false) return;
 
         g_byGuardAI = 0;
-        g_iMp3PlayTime = 0;
-        //        SetBattleCastleStart ( false );
+       g_iMp3PlayTime = 0;
+        //SetBattleCastleStart ( false );
 
         vec3_t Angle, Position;
-        Vector(0.f, 0.f, 0.f, Angle);
-        Vector(0.f, 0.f, 270.f, Position);
-
-        SocketClient->ToGameServer()->SendGuildLogoOfCastleOwnerRequest();
-
-        OpenMonsterModel(77);
-
-        Position[0] = 65 * TERRAIN_SCALE; Position[1] = 113 * TERRAIN_SCALE;
-        CreateObject(MODEL_MONSTER01 + 77, Position, Angle);
-
-        Position[0] = 71 * TERRAIN_SCALE; Position[1] = 113 * TERRAIN_SCALE;
-        CreateObject(MODEL_MONSTER01 + 77, Position, Angle);
-
-        Position[0] = 91 * TERRAIN_SCALE; Position[1] = 113 * TERRAIN_SCALE;
-        CreateObject(MODEL_MONSTER01 + 77, Position, Angle);
-
-        Position[0] = 118 * TERRAIN_SCALE; Position[1] = 113 * TERRAIN_SCALE;
-        CreateObject(MODEL_MONSTER01 + 77, Position, Angle);
-
-        Position[0] = 123 * TERRAIN_SCALE; Position[1] = 113 * TERRAIN_SCALE;
+      Vector ( 0.f, 0.f, 0.f, Angle );
+
+        SendGetCastleGuildMark ();
```

Comment:
This method was replaced by `SocketClient->ToGameServer()->SendGuildLogoOfCastleOwnerRequest();` in my source